_docs/mb-favorite-posts.md

```diff
@@ -101,9 +101,31 @@ To delete a post from the favorite list, just click the icon on the right of the
 
 ## Shortcodes
 
-The plugin provides 2 shortcodes:
-
-- `[mbfp-button]`: to show the "Add to favorite" button, and
-- `[mbfp-posts]`: to show the list of favorite posts
-
-Both shortcodes doesn't have any attributes. They use the settings from the plugin settings page. You can put them anywhere on the website (sidebar, page content, etc.).
+The plugin provides 3 shortcodes:
+You can put them anywhere on the website (sidebar, page content, etc.).
+
+- `[mbfp-posts]`: to show the list of favorite objects
+
+- `[mbfp-button]`: to show the "Add to favorite" button
+
+```php
+[mbfp-button id="custom_id" class="custom_class" add="add_text" added="added_text" show_count=1]
+```
+It accepts the following parameters:
+Name|Description
+---|---
+`id`|Custom ID name. Optional. If missed, the shortcode will get the current object ID.
+`class`|Custom class name. Optional.
+`add`|Custom text if object wasn't add to favorite. Optional. If missed, the shortcode will get from mb-favorite-post plugin settings.
+`added`|Custom text if object was added to favorite. Optional. If missed, the shortcode will get from mb-favorite-post plugin settings.
```

Comment:
Em bỏ cái `mb-favorite-post` đi. Viết mô tả cho user mà. VD:

`add`|Custom button text when the post isn't added to favorites. Optional. If missed, the shortcode will get the value from the plugin settings.
`added`|Custom button text when the post is added to favorites. Optional. If missed, the shortcode will get the value from the plugin settings.